doc/rst/language/spec/conversions.rst

```diff
@@ -218,54 +175,235 @@ type ``C`` is allowed when ``D`` is a subclass of ``C``.
 
 Any combination of these three conversions is allowed.
 
+.. _Subtype_Arg_Conversions:
 .. _Implicit_Type_Arg_Conversions:
+.. _Implicit_Generic_Type_Conversions:
 
-Implicit Type Argument Conversions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Implicit Subtype Conversions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Right, the class conversions section has a lot of details about the owned/shared/borrowed/generic management/nilable or not issues. I didn't want all of that to interfere with the explanation of what subtype conversions are (since these apply for type arguments and the class conversions include some that are not "subtype").

Reply:
Anyway for now I am inclined to leave these sections separate and not cross-linking (mostly because I am getting tired of working on the conversions chapter and it seems a nice-to-have).